.github/ISSUE_TEMPLATE/troubleshooting-question.md

```diff
@@ -4,4 +4,4 @@ about: Ask a question to solve installation and configuration issues
 
 ---
 
-For troubleshooting, see [https://www.mattermost.org/troubleshoot/](https://www.mattermost.org/troubleshoot/).
+For troubleshooting, see [https://docs.mattermost.com/install/troubleshooting.html/](https://docs.mattermost.com/install/troubleshooting.html).
```

Comment:
@cwarnermm should the `/` at the end of [https://docs.mattermost.com/install/troubleshooting.html/] be removed?  It's not in the URL in line 9...

Reply:
I *think* the trailing slash is optional on these URLs. @hmhealey - Are you able to confirm my assumption?

Reply:
```suggestion
For troubleshooting, see [https://docs.mattermost.com/install/troubleshooting.html](https://docs.mattermost.com/install/troubleshooting.html).
```